src/components/common.js

```diff
@@ -33,19 +33,19 @@ export const Clickable = ({ as = 'div', disabled, tooltip, tooltipSide, onClick,
   }
 }
 
-const linkProps = disabled => ({
+const linkProps = (disabled, color = colors.green) => ({
```

Comment:
Still defaults to green, but can select other colors if appropriate for rendering agains non-white backgrounds.

Reply:
FYI I changed this to allow specifying a known color `variant`, rather than an arbitrary color.